server/lib/mixins/matchMixin.js

```diff
@@ -583,6 +584,68 @@ const addPatient = (clientID, patientsBundle, callback) => {
       operSummary.FHIRConflictsMatches = FHIRConflictsMatches.entry;
       operSummary.ESMatches = ESMatches;
 
+
+      // if there is auto matches  then add a tag
```

Comment:
We instead add an extension that specifies if the match was auto or by human adjudication.